code/modules/mob/living/living.dm

```diff
@@ -724,6 +724,7 @@
 	if(layer == initial(layer)) //to avoid things like hiding larvas.
 		layer = LYING_MOB_LAYER //so mob lying always appear behind standing mobs
 	add_traits(list(TRAIT_UI_BLOCKED, TRAIT_PULL_BLOCKED, TRAIT_UNDENSE), LYING_DOWN_TRAIT)
+	playsound(src, SFX_BODYFALL, 50, TRUE)
```

Comment:
This doesn't play the sound twice if someone gets shoved over, right?

Reply:
Oh and does it get played if you buckle to something which requires lying down (beds)? Shouldn't if so